cf-psb/pom.xml

```diff
@@ -32,11 +32,6 @@
 
     <repositories>
         <!-- must repeat repository on all root poms so maven can locate the ocopea parent pom -->
```

Comment:
Remove this comment if you have deleted the repo.

Reply:
@vjtirumalai updated